docs/GettingStarted.md

```diff
@@ -249,7 +247,7 @@ It might not always be possible to wrap `Datadog.tracer.trace` around a block of
 To trace these operations, you can trace code asynchronously by calling `Datadog.tracer.trace` without a block:
 
 ```ruby
-# Some instrumentation framework calls this after an event began and finished...
+# Some instrumentation framework calls this after an event begins and finishes...
```

Comment:
This change may not be accurate; I think the original comment means to say `this is called after an event finishes` (that is only once) whereas `after an event begins and finishes` suggests it is called twice.

Reply:
I updated it to `after an event finishes`